src/Sulu/Bundle/SecurityBundle/Tests/Unit/EventListener/SuluSecurityListenerTest.php

```diff
@@ -179,14 +210,10 @@ public function testCallableControllerPermission($method, $permission)
         $request->getMethod()->willReturn($method);
         $request->get('id')->willReturn('1');
 
-        $controller = $this->prophesize(SecuredControllerInterface::class);
-        $controller->getSecurityContext()->willReturn('security.context');
-        $controller->getLocale(Argument::any())->willReturn('de');
+        $controller = new TestCallableClass();
+        $controllerEvent = $this->createControllerEvent($controller, $request->reveal());
 
-        $this->filterControllerEvent->getRequest()->willReturn($request->reveal());
-        $this->filterControllerEvent->getController()->willReturn($controller->reveal());
```

Comment:
But in that case it was also not a callable class, or was it? Is that on the right handside a different test now?

Reply:
Before it was not really checked by symfony if the given class is a real callable. If it was a object it was handled as a callable and set as `_controller` attribute, but in symfony 5 they added the `callable` typehint so now it need to be real callable which implements the `__invoke` class or a `['class', 'method']` is given but this test case is for the `__invoke` #5152